src/ProductConstructionService/ProductConstructionService.Api/ProductConstructionService.Api.csproj

```diff
@@ -16,6 +16,9 @@
     <PackageReference Include="Swashbuckle.AspNetCore" />
     <PackageReference Include="Azure.Identity" />
     <PackageReference Include="Azure.Extensions.AspNetCore.Configuration.Secrets" />
+    <PackageReference Include="Aspire.Azure.Storage.Queues" />
+	<!-- Health check for the Azure Storage Queue fails if this is not pinned to 7.0.0. Might not be needed when the whole repo gets updated to .NET 8 -->
```

Comment:
I don't follow? The repo already uses 8.0 SDK to build and the PCS is even targeting 8.0? What is left here?

Reply:
Without the explicit reference, the app was using `Microsoft.Bcl.AsyncInterfaces` 6.0.0, I checked the deps.json. This made the health check fail, as the library it was using was looking for `Microsoft.Bcl.AsyncInterfaces` 7.0.0

Reply:
Yeah, no I get that we want the pin but I don't understand the comment "Might not be needed when the whole repo gets updated to .NET 8" as that one doesn't make sense to me

Reply:
Yup, I guess you are right, it doesn't really make sense, I'll fix the comment